source/Purchases.brs

```diff
@@ -1027,7 +1027,7 @@ function _InternalPurchases(o = {} as object) as object
             end for
             activeSubscriptions = []
             for each entry in allExpirationDatesByProduct.Items()
-                if entry.value.asSeconds() > requestDate.asSeconds()
+                if entry.value <> invalid and entry.value.asSeconds() > requestDate.asSeconds()
```

Comment:
This should never happen because `allExpirationDatesByProduct` contains only subscriptions, but better to be safe